Dockerfile.dev

```diff
@@ -1,21 +1,13 @@
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.9
+FROM python:3.10
```

Comment:
We cannot use Python 3.10 as we have to support 3.9 at this point.

Reply:
Got it.